src/videotile/VideoTileState.ts

```diff
@@ -5,25 +5,108 @@
+   */
   tileId: number | null = null;
+
+  /**
+   * Indication of whether tile is associated with local video.
+   */
   localTile: boolean = false;
+
+  /**
+   * Indication of whether local tile has started to play, if this [[VideoTileState]] is to be associated with local tile.
+   */
   localTileStarted: boolean = false;
+
+  /**
+   * Indication of whether the tile has content-sharing video.
+   */
   isContent: boolean = false;
+
+  /**
+   * Indication of whether the tile has active video stream.
+   */
   active: boolean = false;
+
+  /**
+   * Indication of whether the tile has paused video stream.
+   */
   paused: boolean = false;
+
+  /**
+   * Indication of whether the remote video is paused at publishing attendee. This field is not supported.
+   */
   poorConnection: boolean = false;
+
+  /**
+   * The attendee id associated with the [[VideoTile]].
+   */
   boundAttendeeId: string | null = null;
+
+  /**
+   * The user id associated with the [[VideoTile]].
+   */
   boundExternalUserId: string | null = null;
+
+  /**
+   * The video stream bound with the [[VideoTile]].
+   */
   boundVideoStream: MediaStream | null = null;
+
+  /**
+   * The HTMLVideoElement bound with the [[VideoTile]].
+   */
   boundVideoElement: HTMLVideoElement | null = null;
+
+  /**
+   * The nameplate for the [[VideoTile]]. SDK users should use boundExternalUserId for user id instead of this field.
+   */
   nameplate: string | null = null;
+
+  /**
+   * The intrinsic width of the video stream upon binding with the [[VideoTile]].
+   * Video stream intrinsic width could change and developers should use HTMLVideoElement listener for actual intrinsic width.
+   */
   videoStreamContentWidth: number | null = null;
+
+  /**
+   * The intrinsic height of the video stream upon binding with the [[VideoTile]]
+   * Video stream intrinsic height could change and developers should use HTMLVideoElement listener for actual intrinsic height.
+   */
   videoStreamContentHeight: number | null = null;
+
+  /**
+   * The CSS width in pixel of the HTMLVideoElement upon binding with the [[VideoTile]].
+   */
   videoElementCSSWidthPixels: number | null = null;
+
+  /**
+   * The CSS height in pixel of the HTMLVideoElement upon binding with the [[VideoTile]].
+   */
   videoElementCSSHeightPixels: number | null = null;
+
+  /**
+   * The device pixel ratio of the current display monitor.
+   */
   devicePixelRatio: number = 0;
+
+  /**
+   * The physical width in pixel of the HTMLVideoElement upon binding with the [[VideoTile]].
+   */
   videoElementPhysicalWidthPixels: number | null = null;
+
+  /**
+   * The physical height in pixel of the HTMLVideoElement upon binding with the [[VideoTile]].
+   */
   videoElementPhysicalHeightPixels: number | null = null;
+
+  /**
+   * The unique identifier for the video stream bound with the [[VideoTile]].
```

Comment:
Nit, separate lines or do we want to justify them better?

Reply:
What do you mean by separate lines ?

Reply:
The formatting looked strange - first line shorter than the others. I wont block the PR on this - as it may rendered correctly. I would rather have the right content out ;)

Reply:
Good catch. I rephrased the words